### PR TITLE
Homework6: Natallia Rogova

### PR DESCRIPTION
**Pull request by nrogova**

### CODE REVIEW

File: src/homework6_task1.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import module_to_apply as m_a
+""" Task 6.1 Оформите решение задач из прошлых домашних работ в функции.
+Напишите функцию runner.
+runner1() – все фукнции вызываются по очереди
+runner2(‘gen_numbers’) – вызывается только функцию gen_numbers.
+runner3(‘func’, ‘func1’...) - вызывает все переданные функции
+"""
+
+all_functions = ('wierd_func', "arifm_func", 'devision_func',
```
Comment (belovmd) [inline src/homework6_task1.py:9]: we can use smth like [func for func in dir(module) if callable(func)]

File: src/homework6_task4.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        with open('top250_movies.txt', 'a') as top:
+            top.write(top_str + "\n")
+
+    for el in b:
+        y = el[-1].strip('()/I')
+        with open('years.txt', 'a') as years:
+            years.write(y + "\n")
+
+    for el in b:
+        r = el[2]
```
Comment (belovmd) [inline src/homework6_task4.py:28]: unclear namings

**Review by belovmd — COMMENTED** [review]
6 
missed tasks, unoptimal solutions

File: src/homework6_task4.py (hunk 2 above)
Comment (nrogova) [reply src/homework6_task4.py:28]: OMG, sorry. It was my trial version and I forgot to change variables.

**Pull request closed without merging**